EventVisualisation/Base/include/EventVisualisationBase/VisualisationConstants.h

```diff
@@ -0,0 +1,60 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    VisualisationConstants.h
+/// \author  Jeremi Niedziela
+///
+
+#ifndef ALICE_O2_EVENTVISUALISATION_BASE_VISUALISATIONCONSTANTS_H
+#define ALICE_O2_EVENTVISUALISATION_BASE_VISUALISATIONCONSTANTS_H
+
+#include <string>
+
+namespace o2  {
+namespace EventVisualisation {
+
+enum EDetectors{
+  ACO,
+  EMC,
+  HMP,
+  MCH,
+  PHS,
+  RPH,
+  SDD,
+  SPD,
+  SSD,
+  TOF,
+  TPC,
+  TRD,
+  RND,
+  Ndetectors
+};
+  
+const std::string gDetectorNames[Ndetectors] = {
+  "ACO",
+  "EMC",
+  "HMP",
+  "MCH",
+  "PHS",
+  "RPH",
+  "SDD",
+  "SPD",
+  "SSD",
+  "TOF",
+  "TPC",
+  "TRD",
+  "RND"
+};
```

Comment:
@shahor02 also started writing a class for the detector ID
Detectors/Base/include/DetectorsBase/DetID.h
Would it be possible to just have one class in O2 for this in order to avoid the mess we have in AliRoot where these definitions pop up in several places?
Not sure if this could be moved to a more general place.

Reply:
@jniedzie @wiechula : sure, we should not proliferate detector definitions. Moreover that detector names in this header does not correspond to Run3 detectors.
For the directory: is not Detector/Base general enough?

Reply:
Not really (at least not completely), because in case of visualisation we use it to define which geometry volumes and which data types should be drawn. This does not always have one-to-one correspondence to the detectors, for example we need a special geometry for R-Phi projection, which behaves as any other detector geometry, but is not a detector ("RPH" constant). Or we may have a data visualisation type which is not linked to a specific detector (for example "RND" is a random event). 

I think that the best I can do it to rename these enums and arrays to better reflect what they are for.

Reply:
I would also rename the variable to something less misleading. `gDetectorNames`, `EDetectors` make you think this is actual detector names, not what you describe. How about `gVisualisationGroups` or something like that?

Reply:
Absolutely, I renamed them in the latest commit.

Reply:
@jniedzie : Still, for real detectors it is better to use the same definitions, both in names and identifiers (e.g. this would ensure that the bit masks made of detIDs are compatible). In the DetID class version which I will ask to merge in a couple of days (https://github.com/shahor02/AliceO2/blob/86f33f3f8280df8706c2e21549455e8d135ca4cb/Detectors/Base/include/DetectorsBase/DetID.h)  everything is constexpr, so you can alias the IDs and names real detectors in your lists to DetID, e.g. DetID::ITS and DetID(DetID::ITS).getName() respectively, and continue for visualization specific IDs as e.g. RPH = DetID::nDetectos, ....
Although I agree with Jens that mixing the detectors and projections can be misleading.
@wiechula : for the DAQ namings: we still have to marry the DetID definitions with DataOrigin defined in the O2/DataFormats/Headers/include/Headers/DataHeader.h

Reply:
I agree that mixing detectors with projections etc. is in general not a good idea. The problem is that in the visualisation we may have:
- objects specific to detectors, e.g. hits in ITS (one detector -> one object),
- "common" objects, originating from the information of several detectors, e.g. reconstructed tracks (many detectors -> one object),
- different geometries or ways or representing data for single detector, e.g. different TPC geometries for 3D view and R-Phi projection (one detector -> many objects).

Because of that I think that the proposal of @ktf is the best solution (at least for the time being), so that I create "Visualisation Groups" which define identifiers for groups of visualisation objects, which may correspond to the detector, but does not necessarily have to.